### PR TITLE
feat(icons): add new icons

### PR DESCRIPTION
**Pull request by dyhalmeida**
## Infos

[Task](https://juntossomosmais.monday.com/boards/5337789717/views/117729068/pulses/6445252128)

#### What is being delivered?

- Add new icons
- Generate new icons types

#### What impacts?

- Icons package and component

### CODE REVIEW

**Review by felipefialho — APPROVED** [review]
@dyhalmeida This is a `feat` task, not a `chore` task

File: packages/icons/svg/custom/outline-person-blocked.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_1_10729)">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14 3C11.79 3 10 4.79 10 7C10 7.13643 10.0172 7.26712 10.0343 7.39642L10.0343 7.39643L10.0343 7.39655L10.0343 7.39667L10.0343 7.39679C10.0398 7.43795 10.0452 7.47898 10.05 7.52L13.48 10.95C13.5211 10.9548 13.5623 10.9603 13.6036 10.9657C13.7329 10.9828 13.8636 11 14 11C16.21 11 18 9.21 18 7C18 4.79 16.21 3 14 3ZM14 5C15.1 5 16 5.9 16 7C16 7.99 15.27 8.82 14.33 8.97L12.02 6.66C12.19 5.72 13.01 5 14 5ZM21.53 19L15.69 13.16C18.44 13.5 22 14.86 22 17V19H21.53ZM12.01 15.13L13.88 17H8C8.08 16.76 8.88 15.99 10.91 15.43L12.01 15.13ZM2 5.12L3.41 3.71L21.29 21.59L19.88 23L15.88 19H6V17C6 15.3 8.19 14.11 10.39 13.51L7.88 11L3 6.12L2 5.12Z" fill="#1F2328"/>
```
Comment (igorwessel) [inline packages/icons/svg/custom/outline-person-blocked.svg:3]: issue: we shoild remove fill

File: packages/icons/svg/mdi/search-off.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_1_6564)">
+<path d="M16.5 14H15.71L15.43 13.73C16.41 12.59 17 11.11 17 9.5C17 5.91 14.09 3 10.5 3C7.08003 3 4.28003 5.64 4.03003 9H6.05003C6.30003 6.75 8.18003 5 10.5 5C12.99 5 15 7.01 15 9.5C15 11.99 12.99 14 10.5 14C10.33 14 10.17 13.97 10 13.95V15.97C10.17 15.99 10.33 16 10.5 16C12.11 16 13.59 15.41 14.73 14.43L15 14.71V15.5L20 20.49L21.49 19L16.5 14Z" fill="#1F2328"/>
+<path d="M7.47007 10.82L5.00007 13.29L2.53007 10.82L1.82007 11.53L4.29007 14L1.82007 16.47L2.53007 17.18L5.00007 14.71L7.47007 17.18L8.18007 16.47L5.71007 14L8.18007 11.53L7.47007 10.82Z" fill="#1F2328"/>
```
Comment (igorwessel) [inline packages/icons/svg/mdi/search-off.svg:4]: issue: we should remove fill attribute

**Review by igorwessel — CHANGES_REQUESTED** [review]

File: packages/icons/svg/custom/outline-person-blocked.svg (hunk 1 above)
Comment (igorwessel) [reply packages/icons/svg/custom/outline-person-blocked.svg:3]: If we don't remove it we won't be able to style the icon color.

File: packages/icons/svg/mdi/search-off.svg (hunk 2 above)
Comment (igorwessel) [reply packages/icons/svg/mdi/search-off.svg:4]: If we don't remove it we won't be able to style the icon color.

**Review by igorwessel — APPROVED** [review]

**Review by mauriciomutte — CHANGES_REQUESTED** [review]
block: I think both icons should be mdi

**Review by igorwessel — APPROVED** [review]

**Review by mauriciomutte — APPROVED** [review]

**Review by gabrielduete — APPROVED** [review]

**Pull request merged**